modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -459,7 +492,9 @@ var CBIWifiFrequencyValue = form.Value.extend({
 		if (hwval != null) {
 			this.useBandOption = false;
 
-			if (/a/.test(hwval))
+			if (/be/.test(mode.value))
+				band.value = '6g';
+			else if (/ax/.test(mode.value))
```

Comment:
I don't know whether we should nitpick here, since ax technically has 6GHz band also. Do you understand what this function area takes care of?

Reply:
this function is initialising  the various HTML form elements related to configuring the wireless settings. I would suggest to keep it as is.